examples/postgresql-typeorm/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
idk where this file comes from but recommended settings would be the same as in https://github.com/talentlessguy/tinyhttp/blob/8f08d49e126aa423b48f8d76cc916610fada99d1/examples/typescript/tsconfig.json

it shouldn't target ES5, neither has to include lib of ES5 and ES6